packages/react/src/details/docs/overview.mdx

```diff
@@ -8,7 +8,7 @@ relatedComponents: ['accordion', 'tabs']
 ---
 
 ```jsx live
-<Details label="How does the 120-day processing time work?" iconBefore>
+<Details label="Details label?" iconBefore>
```

Comment:
```suggestion
<Details label="How does the 120-day processing time work?" iconBefore>
```

What do you think about keeping a long label here to show consumers what it would look like and to highlight responsive behaviour?

Reply:
I'd prefer to use something like "This is an example of a long label"

Reply:
Cool, sounds good!

```suggestion
<Details label="This is an example of a long label" iconBefore>
```